raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -50,30 +56,36 @@ library ChannelManagerLibrary {
     /// @notice newChannel(address, uint) to create a new payment channel between two parties
     /// @dev Create a new channel between two parties
     /// @param partner (address) the address of the partner
-    /// @param settleTimeout (uint) the settleTimeout in blocks
+    /// @param settle_timeout (uint) the settle timeout in blocks
     /// @return (address) the address of the NettingChannelContract.
-    function newChannel(Data storage self, address partner, uint settleTimeout) returns (address) {
-        address channelAddress;
+    function newChannel(
+        Data storage self,
+        address partner,
+        uint settle_timeout
+    )
+        returns (address)
```

Comment:
The `returns` should be right after the parentheses, it can be mistaken for the body of the function